### PR TITLE
[ui] Fix scroll behavior, missing tabs when using new runs feed tables

### PR DESCRIPTION
**Pull request by bengotow**

### CODE REVIEW

File: js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedTable.tsx [hunk 1]
```diff
@@ -195,7 +195,7 @@ export const RunsFeedTableWithFilters = ({
   filter: RunsFilter;
 } & Pick<
   RunsFeedTableProps,
-  'actionBarComponents' | 'belowActionBarComponents' | 'emptyState' | 'hideTags'
+  'actionBarComponents' | 'belowActionBarComponents' | 'emptyState' | 'hideTags' | 'scroll'
```
Comment (salazarm) [inline js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedTable.tsx:198]: you're not actually passing scroll on line 221, it's hardcoded to false still

Comment (bengotow) [reply js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedTable.tsx:198]: ahh It's overridden in `...rest` actually, but i'll make it explicit

Comment (bengotow) [reply js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedTable.tsx:199]: Updated this to destructure it and apply the default value as `?? false`

Comment (salazarm) [reply js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedTable.tsx:199]: oops, missed that

**Review by salazarm — APPROVED** [review]

File: js_modules/dagster-ui/packages/ui-core/src/schedules/ScheduleRoot.tsx [hunk 2]
```diff
@@ -133,7 +133,7 @@ export const ScheduleRoot = (props: Props) => {
                 tabs={tabs}
               />
             ) : (
-              <RunsFeedTableWithFilters filter={runsFilter} />
+              <RunsFeedTableWithFilters filter={runsFilter} actionBarComponents={tabs} />
```
Comment (bengotow) [inline js_modules/dagster-ui/packages/ui-core/src/schedules/ScheduleRoot.tsx:136]: This was causing these tabs to disappear when you switched to Run History

image

File: js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedTable.tsx (hunk 1 above)
Comment (salazarm) [reply js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedTable.tsx:199]: I think what you had before is fine

**Review by salazarm — APPROVED** [review]

**Pull request merged**